.github/workflows/main.yml

```diff
@@ -5,7 +5,7 @@ on:
     paths-ignore:
       - '**.md'
     branches:
-      - master
+      - '**'
```

Comment:
I guess this was needed for testing on different branches in your fork, but that has no place in a feature PR.